CovtracerCheck.R

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env Rscript
+
```

Comment:
Maybe there should be a short description what this R script is doing

Reply:
added